media-api/app/lib/ImageResponse.scala

```diff
@@ -24,11 +24,18 @@ object ImageResponse {
 
   def fileMetaDataUri(id: String) = URI.create(s"${Config.rootUri}/images/$id/fileMetadata")
 
+  def isPhotographerCategory[T <: UsageRights](usageRights: T) =
```

Comment:
[Hopefully going to get a new trait in which would help with this](https://github.com/guardian/grid/pull/1205/files#diff-b5fd3ed3a620cfe97ed5b6a40a586bd3R80).